src/shared/getPrice.js

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react";
+import { api } from "./api.js";
+
+export function getPriceInPLN(pricePerNightInEURCent) {
```

Comment:
hook name should always start with `use`

Reply:
done